cli/consts.py

```diff
@@ -48,6 +48,8 @@
 
 DEFAULT_CYCODE_API_URL = "https://api.cycode.com"
 DEFAULT_CYCODE_APP_URL = "https://app.cycode.com"
+DEV_SCAN_CYCODE_APP_URL = "http://localhost:5004"
+DEV_DETECTION_CYCODE_APP_URL = "http://localhost:5016"
```

Comment:
I think you should create config_dev.py file and put there all the related dev ENV vars

Reply:
Fixed